en/intro/conventions.rst

```diff
@@ -142,6 +142,18 @@ Now that you've been introduced to CakePHP's fundamentals, you might try a run
 through the :doc:`/tutorials-and-examples/bookmarks/intro` to see how things fit
 together.
 
+Core Naming Conventions
```

Comment:
Would it be a good idea to try and standardize how we use those 3 words so that they are less ambiguous? For example we could use 'column' only when referring to database columns, 'property' when referring to ORM properties, and 'field' in other places?